airbyte-integrations/connectors/source-github/source_github/source.py

```diff
@@ -65,28 +70,41 @@ def _generate_repositories(self, config: Mapping[str, Any], authenticator: Token
             repos = Repositories(authenticator=authenticator, organizations=organizations)
             for stream in repos.stream_slices(sync_mode=SyncMode.full_refresh):
                 repositories_list += [
-                    repository["full_name"] for repository in repos.read_records(sync_mode=SyncMode.full_refresh, stream_slice=stream)
+                    r["full_name"] for r in repos.read_records(sync_mode=SyncMode.full_refresh, stream_slice=stream)
                 ]
         if repositories:
             repositories_list += repositories
 
         return list(set(repositories_list))
 
+    @staticmethod
+    def _get_authenticator(token: str):
+        if TOKEN_SEPARATOR not in token:
```

Comment:
do we need a special edge case for this? multiple auth should still work with one token right?